tests/default_vars.sh

```diff
@@ -449,3 +453,106 @@ export_35d_run ()
 export CNTL_DIR=""
 export LIST_FILES=""
 }
+export_datm ()
+{
+export FV3=false
+export S2S=false
```

Comment:
Why S2S=false.
If FV3=false and S2S=false, what are we doing with DATM=true?

Reply:
Bin, Do you want to set S2S=true?

Reply:
For FV3 tests, FV3=true, S2S=false, DATM=false
For S2S tests, FV3=true, S2S=true, DATM=false
For DATM tests, FV3=false, S2S=false, DATM=true

Please see run_test.sh.